misk/src/main/kotlin/misk/eventrouter/KubernetesClusterConnector.kt

```diff
@@ -35,13 +35,16 @@ internal class KubernetesClusterConnector : ClusterConnector {
   private var lastReceivedMessage: Instant? = null
 
   fun healthStatus(): HealthStatus {
-    if (lastReceivedMessage == null) return HealthStatus.unhealthy("I've never received a message")
+    if (lastReceivedMessage == null) {
+      return HealthStatus.unhealthy("k8s: I've never received a message")
+    }
     val sinceLastReceived = Duration.between(lastReceivedMessage, clock.instant()).seconds
     if (sinceLastReceived > config.kubernetes_read_timeout + config.kubernetes_connect_timeout) {
       return HealthStatus.unhealthy(
-          "I haven't received an update in $sinceLastReceived seconds.")
+          "k8s: I haven't received an update in $sinceLastReceived seconds.")
     }
-    return HealthStatus.healthy("I received a message $sinceLastReceived seconds ago.")
+    return HealthStatus.healthy(
+        "k8s: I received a message $sinceLastReceived seconds ago.")
```

Comment:
nit: why the `k8s` prefix? The logger name should help you search for these log message

Reply:
The logger/class name isn't stored in `HealthStatus` or `HealthCheck`.  I could add that.